src/FifoMap.js

```diff
@@ -49,6 +49,12 @@ export class FifoMap {
     }
   }
 
+  deleteMany(keys) {
+    for (var i = 0; i < keys.length; i++) {
```

Comment:
🟢 is there a reason not to use `for (const key of keys)` syntax? :)

Reply:
yeah, according to fastify folks, `var` is slightly faster in loops. for a very low-level library every bit helps :D

Reply:
oh wow! good to know 😁